<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credits</class>
 <widget class="QMainWindow" name="Credits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Credits</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="creditLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="studentLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a Student</string>
    </property>
   </widget>
   <widget class="QListView" name="studentListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="creditDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
